README.md

```diff
@@ -64,6 +64,35 @@ to the `require` section of your `composer.json`.
 Usage
 -----
 
+Configure socialNetworks in config file
+
+```php
+    'components' => [
+    // ...
+        'socialShare' => [
+            'class' => \ymaker\social\share\configurators\Configurator::class,
+            'socialNetworks' => [
+                'vkontakte' => [
+                    'class' => \ymaker\social\share\drivers\Vkontakte::class,
+                ],
+                'facebook' => [
+                    'class' => \ymaker\social\share\drivers\Facebook::class,
+                ],
+                'odnoklasniki' => [
+                    'class' => \ymaker\social\share\drivers\Odnoklassniki::class,
+                ],
+                'viber' => [
```

Comment:
Can you remove `viber` and `whatsapp`? I think 3 social networks will be enough.

Reply:
Ok.